src/dotnet/Chat.Service/Chats.cs

```diff
@@ -181,18 +185,40 @@ public virtual async Task<Chat> OnChange(Chats_Change command, CancellationToken
             changeCommand = changeCommand with {
                 OwnerId = account.Id,
             };
+            var chatDiff = changeCommand.Change.Create.Value;
```

Comment:
there was `change.IsCreate(out var ...)` which you can use in if block above

Reply:
done